seed/serializers/properties.py

```diff
@@ -177,6 +190,43 @@ def to_representation(self, data):
 
         return result
 
+    def _collapse_pint_value(self, property_state, field_name):
```

Comment:
I was just messing with this today in order to fix a bug on a production system. I think we'll want to converge the two together. In short instead of creating a SerializerMethodField, I created a new serializer.Field.

Reply:
@rgm -- see this PR https://github.com/SEED-platform/seed/pull/1542/files. I was going to merge this into master after some more testing, then eventually pull it up to develop and all dev-branches.